articles/quickstart/native/flutter/files/main.md

```diff
@@ -0,0 +1,19 @@
+---
+name: main.dart
+language: dart
+---
+
+```dart
+// Instantiate the SDK once in your app
+final auth0 = Auth('${account.namespace}', '${account.clientId}');
+
+// Redirect to Auth0 Universal Login for authentication
+final result = await auth0.webAuthentication.login();
+final token = result.accessToken
+
+final name = result.userProfile.name;
+final picture = result.userProfile.pictureURL;
+
+// Redirect to the logout endpoint on Auth0
+await auth0.webAuthentication.logout();
```

Comment:
Should we show this code inside a button handler or something that gives a little more context? 
E.g.:

<img width="578" alt="Screen Shot 2022-07-06 at 19 14 20" src="https://user-images.githubusercontent.com/5055789/177652444-50212bc4-428d-4a63-85c7-a7430ca3db21.png">

Reply:
Ideally, the snippets should form kind of a barebones sample app, and include all the code changes necessary to perform login, logout, and access the user profile information, and do it all in context.

<img width="618" alt="Screen Shot 2022-07-06 at 19 17 04" src="https://user-images.githubusercontent.com/5055789/177652905-10cba6d0-ba23-4b8c-88dd-7384a26dbd68.png">

Reply:
I added some new snippets in 2cdc27402 - what do you think?

![image](https://user-images.githubusercontent.com/766403/177813529-c9309602-22cf-4803-ae8f-7d1406ffe369.png)

![image](https://user-images.githubusercontent.com/766403/177813640-d336c0d7-02e1-42d8-b0d2-32531b25927e.png)

Reply:
Much better. Just left a couple of comments, in particular one about how we can simplify it a bit more.